src/client-joo/target_table.ml

```diff
@@ -563,15 +563,15 @@ module Html = struct
                 a_class ["label"; label];
                 a_onmouseover (fun ev ->
                     let mx, my =
-                      Js.Optdef.case ev##.toElement
+                      Js.Optdef.case ev##.relatedTarget
                         (fun () ->
-                           Log.(s "toElement undefined !!" @ error);
+                           Log.(s "relatedTarget undefined !!" @ error);
                            (200, 200))
                         (fun eltopt ->
                            Js.Opt.case eltopt
                              (fun () ->
-                                Log.(s "toElement defined but null!!" @ error);
-                                (200, 200))
+                                Log.(s "relatedTarget defined but null!!" @ error);
+                                (400, 400))
```

Comment:
Hah, alright :)

On Tue, Nov 3, 2015 at 5:30 PM, Sebastien Mondet <notifications@github.com>
wrote:

> In src/client-joo/target_table.ml
> <https://github.com/hammerlab/ketrew/pull/243#discussion_r43816766>:
>
> >                             (200, 200))
> >                          (fun eltopt ->
> >                             Js.Opt.case eltopt
> >                               (fun () ->
> > -                                Log.(s "toElement defined but null!!" @ error);
> > -                                (200, 200))
> > +                                Log.(s "relatedTarget defined but null!!" @ error);
> > +                                (400, 400))
>
> Ah yeah :) It's for next time this breaks, I made them different so that
> it easier to guess which one of undefined or null it is (even if looking
> at the console output is enough).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hammerlab/ketrew/pull/243/files#r43816766>.
>